src/js/components/OptionalContainerVolumesComponent.jsx

```diff
@@ -0,0 +1,146 @@
+import classNames from "classnames";
+import React from "react/addons";
+
+import DuplicableRowControls from "../components/DuplicableRowControls";
+import DuplicableRowsMixin from "../mixins/DuplicableRowsMixin";
+import FormGroupComponent from "../components/FormGroupComponent";
+
+import ContainerConstants from "../constants/ContainerConstants";
+
+var OptionalVolumesComponent = React.createClass({
+  displayName: "OptionalContainerVolumesComponent",
```

Comment:
I find the naming confusing, there is also a OptionalVolumesComponent.
What is what? Does it makes sense to combine these?